Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -774,6 +895,8 @@ TextureCacheBase::TCacheEntry* TextureCacheBase::Load(const u32 stage)
   config.levels = texLevels;
   config.format = hires_tex ? hires_tex->GetFormat() : AbstractTextureFormat::RGBA8;
 
+  ArbitraryMipmapDetector arbitrary_mip_detector;
```

Comment:
This one looks very far apart from the actual usage some 60 lines further down. Can you move it closer to its usage?

Reply:
That's just before the scope where it is used starts, so it would be hard to move it down.